paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -148,19 +149,25 @@ internal fun Wallet(
     val googlePayButtonState by viewModel.googlePayButtonState.collectAsState(initial = null)
     val buttonsEnabled by viewModel.buttonsEnabled.collectAsState(initial = false)
 
+    val isDark = !MaterialTheme.colors.surface.shouldUseDarkDynamicColor()
+
     val padding = dimensionResource(R.dimen.stripe_paymentsheet_outer_spacing_horizontal)
 
     if (containerState.shouldShow) {
-        Column(modifier = modifier.padding(horizontal = padding)) {
+        // Remove 4.dp from each side to account for Google pay internal padding
+        // Link button will add 4.dp on each side to account for this
+        Column(modifier = modifier.padding(horizontal = padding - 4.dp)) {
```

Comment:
Do we need this here or can we do it internally in `GooglePayButton`?

Reply:
Google Pay button has internal padding which we cannot remove